test_data/proxy/src/com/intellij/memory/agent/MemoryAgent.java

```diff
@@ -0,0 +1,127 @@
+package com.intellij.memory.agent;
+
+import java.io.File;
+import java.lang.reflect.Array;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.concurrent.Callable;
+
+public class MemoryAgent {
```

Comment:
This class is a part of the public API of our library. I think it worth adding JavaDoc for public methods and describe their contracts, exceptions, return types, etc.